.circleci/config.yml

```diff
@@ -130,6 +131,40 @@ jobs:
           # without it.
           command: bash <(curl -s https://codecov.io/bash) -C "$CIRCLE_SHA1"
 
+  go-test-race:
+    docker:
+      - image: *GOLANG_IMAGE
+    environment:
+      <<: *ENVIRONMENT
+      GOTAGS: "" # No tags for OSS but there are for enterprise
+      # GOMAXPROCS defaults to number of cores on underlying hardware, set
+      # explicitly to avoid OOM issues https://support.circleci.com/hc/en-us/articles/360034684273-common-GoLang-memory-issues
+      GOMAXPROCS: 4
+      # The medium resource class (default) boxes are 2 vCPUs, 4GB RAM
+      # https://circleci.com/docs/2.0/configuration-reference/#docker-executor
+      # but we can run a little over that limit.
+    steps:
+      - checkout
+      - run: *install_gotestsum
```

Comment:
this will conflict with https://github.com/hashicorp/consul/pull/8327 sorry!

Reply:
No worries! Luckily it does not appear to conflict

Reply:
It might not conflict but it won't work properly when merged so you'll have to rebase since `install_gotestsum` doesn't exist anymore in `master`

Reply:
Oh right! The alias is a dash now, not an underscore. I missed that! I'll rebase now